Doc: update for taxi (#2932)
hove-io/navitia: documentation/slate/source/includes/stuff.md

depicting specifically the ridesharing ads that could match the above section
and that could be proposed to the user.

<a name="taxi-stuff"></a>Taxi
-------------------------------------------

<aside class="warning">

>     This feature is only available in `distributed` scenario.

Reviewer: Same here, I'd remove the reference to the scenario.
```suggestion
    This feature is not available on all coverages, as it is dependent on other parameters of the coverage (how journeys are computed).
```

</aside>

> With this mode, your journey may contain taxi sections(fallback or diret path). The journey you will obtain is basically as same as a journey by car. The only difference is that with taxi as fallback mode, a buffer time(marked as waiting, defaulted to 5 min) will appear int the section. The buffer time won't appear if the journey is a direct path. Depending on the calculator, the journey may pick up ways that reserved for taxis on not.

Reviewer: ```suggestion
With this mode, your journey may contain taxi sections(fallback or direct path). The journey you will obtain is basically the same as a journey by car. The only difference is that with taxi as fallback mode, a buffer time (section "waiting", defaulted to 5 min) will appear into the journey. The buffer time won't appear if the journey is a direct path. Depending on the calculator, the journey may pick up ways that are reserved for taxis on not.
```
